tests/e2e/performance/csi_tests/test_pvc_multi_snapshot_performance.py

```diff
@@ -57,6 +61,9 @@ def setup(self):
 
         super(TestPvcMultiSnapshotPerformance, self).setup()
 
+        self.total_creatio_time = 0
```

Comment:
I suggest to rename to creation_time instead of creatio_time ( and refactor all appearances)

Reply:
done. thanks